server/service/src/sync/test/test_data/program_requisition_settings.rs

```diff
@@ -119,6 +119,88 @@ const MASTER_LIST_WITH_PROGRAM_2: (&str, &str) = (
 }"#,
 );
 
+// const MASTER_LIST_WITH_EMPTY_OBJECT_PROGRAM_SETTINGS: (&str, &str) = (
+//     "program_test_3",
+//     r#"{
+//     "ID": "program_test_3",
+//     "description": "Program Test 03",
+//     "date_created": "2017-08-17",
+//     "created_by_user_ID": "0763E2E3053D4C478E1E6B6B03FEC207",
+//     "note": "note 4",
+//     "gets_new_items": false,
```

Comment:
Added this test data in case it could be used to test deserialising of program_requisition_setting - but I think in this case it is OK as the deserialising handles having no program settings in the generate function, and throws a specific error.

Reply:
I think we'll jsut take these out for now. It's only managed on central server so not likely to be messed up to an empty object...